Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -22,16 +23,27 @@ public class Main : ISettingProvider, IPlugin, IReloadable, IPluginI18n, IContex
         private static List<Bookmark> cachedBookmarks = new List<Bookmark>();
 
         private static Settings _settings;
-        
+
         public void Init(PluginInitContext context)
         {
             Main.context = context;
-            
+
             _settings = context.API.LoadSettingJsonStorage<Settings>();
 
-            cachedBookmarks = BookmarkLoader.LoadAllBookmarks(_settings);
+            LoadBookmarksIfEnabled();
+        }
+
+        private static void LoadBookmarksIfEnabled()
+        {
+            if (context.CurrentPluginMetadata.Disabled)
+            {
+                // Don't load or monitor files if disabled
+                // Note: It doesn't start loading or monitoring if enabled later, you need to manually reload data
+                return;
+            }
```

Comment:
We shouldn't call init() when plugin is disabled.

Reply:
Yeah but it's not the point of this PR.

Reply:
> should we call init every time plugin is reenabled?

Technically we should but some plugins may need to update. like if we don't dispose file watchers when disabling plugins, creating new ones after re enabling may lead to some issues.